openbsd/OpenBSDProcessList.c

```diff
@@ -59,7 +60,7 @@ ProcessList* ProcessList_new(UsersTable* usersTable, Hashtable* pidWhiteList, ui
 
    size = sizeof(fscale);
    if (sysctl(fmib, 2, &fscale, &size, NULL, 0) < 0)
-     CRT_fatalError("fscale sysctl call failed");
+     errx(1, "fscale sysctl call failed");
```

Comment:
`sysctl(3)` on OpenBSD seems to have a lot of possible failure conditions; why not use `err(3)` instead?